doc/en/example/simple.rst

```diff
@@ -296,7 +314,9 @@ which will add the string to the test header accordingly::
 You can also return a list of strings which will be considered as several
 lines of information.  You can of course also make the amount of reporting
 information on e.g. the value of ``config.getoption('verbose')`` so that
-you present more information appropriately::
+you present more information appropriately:
```

Comment:
I find this sentence slightly confusing, might be out-of-scope for this PR, but I think we should reword it.